sky/provision/__init__.py

```diff
@@ -4,13 +4,18 @@
 providers supported by SkyPilot need to follow.
 """
 import functools
-import importlib
 import inspect
 from typing import Any, Dict, List, Optional
 
 from sky import sky_logging
 from sky import status_lib
+# These provision.<cloud> modules should never fail even if underlying cloud SDK
+# dependencies are not installed. This is ensured by using sky.adaptors inside
+# these modules, for lazy loading of cloud SDKs.
+from sky.provision import aws
+from sky.provision import azure
 from sky.provision import common
+from sky.provision import gcp
```

Comment:
later we may scan over the directory to import all modules automatically. I am fine with the current implementation though.